tools/gen_addon_readme.rst.jinja

```diff
@@ -6,7 +6,12 @@
 {{ fragments[name] }}
 {% endif %}
 {%- endmacro -%}
+=======
+.. image:: https://odoo-community.org/web/image/4610395-7254fada/OCA_banner_readme_1256x150px.png
```

Comment:
Could we use a nice stable URL here and set up a redirect on the website? In case our website changes we would not want to break existing READMEs.